### PR TITLE
Fixes mvn package to be usable by JDKs greater than 1.8

### PR DESCRIPTION
**Pull request by ryanwilson-wk**
### Story:
Fixes mvn package to be usable by JDKs greater than 1.8

### Acceptance Criteria:
- [ ] At least one InfRe Squad 2 member has reviewed and +1'd
- [ ] Code has been tested and results documented
- [ ] Unit tests have been updated
- [ ] Updates to documentation if necessary
- [ ] Verify and document changes to any other Messaging components
- [ ] Pull request made against the 'develop' branch, not master

### Design Notes:
TODO

### How To Test:
TODO

### My Test Results:
TODO

#### Reviewers:
@Workiva/product2

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
<details>
<summary>(3) Vulnerable direct dependencies were detected</summary>
<li>2 vulns in <code>org.apache.thrift:libthrift < 0.12.0</code> via <code>examples/java/pom.xml</code></li>
<li>2 vulns in <code>org.apache.thrift:libthrift < 0.12.0</code> via <code>lib/java/pom.xml</code></li>
<li>2 vulns in <code>org.apache.thrift:libthrift < 0.12.0</code> via <code>test/integration/java/frugal-integration-test/pom.xml</code></li>
</details>

## Action Items
* Review dependencies for available updates
* See <a href="https://splunk.workiva.net/en-US/app/search/security_vulnerable_dependencies?form.repository=frugal">this Splunk dashboard</a> for more CVE details
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Comment by kevinsookocheff-wf** [comment]
+1

File: lib/java/pom.xml [hunk 1]
```diff
@@ -292,6 +292,9 @@
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-javadoc-plugin</artifactId>
                         <version>3.2.0</version>
+                        <configuration>
+                            <source>8</source>
+                        </configuration>
```
Comment (brettkail-wk) [inline lib/java/pom.xml:297]: For posterity, this is the error I see building with Java 11 (I assume Ryan sees the same):
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.2.0:jar (attach-javadocs) on project frugal: MavenReportException: Error while generating Javadoc:
...
[ERROR] javadoc: error - The code being documented uses modules but the packages defined in https://docs.oracle.com/javase/8/docs/api/ are in the unnamed module.

**Comment by natewoods-wf** [comment]
HOLD until after CVE resolution releases

File: lib/java/pom.xml (hunk 1 above)
Comment (ryanwilson-wk) [reply lib/java/pom.xml:297]: Yep, that's what I saw

**Comment by natewoods-wf** [comment]
With the release of v3.11.0, we should be good on the CVE front

@Workiva/release-management-p for merge

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**